src/webview.js

```diff
@@ -76,7 +87,7 @@ export default class WebView extends React.Component {
 }
 
 const tagPropTypes = {
-  autosize: React.PropTypes.bool,
+  autosize: React.PropTypes.string,
```

Comment:
Same as above. Should probably test that this can be a boolean and not be a string. Although if it's too confusing, I'm okay with it being a union type.